src/js/components/Menu/__tests__/Menu-test.js

```diff
@@ -58,6 +59,31 @@ describe('Menu', () => {
     expect(component.toJSON()).toMatchSnapshot();
   });
 
+  test('gap between icon and label', () => {
+    window.scrollTo = jest.fn();
+    const { container, getByText } = render(
+      <Grommet>
+        <Menu
+          open
+          label="actions"
+          items={[
+            { label: 'Item 1', icon: <svg />, gap: 'xlarge' },
+            { label: 'Item 2' },
+          ]}
+        />
+      </Grommet>,
+    );
+
+    const firstItem = getByText('Item 1');
+    expect(
+      firstItem.getElementsByClassName(
+        'StyledBox__StyledBoxGap-sc-13pk1d4-1 bPCnCu',
```

Comment:
I wonder if this class name is something we can count on with the testing, I think it may change once snapshots classes are updated (once a package is updated foe=r example).
maybe we should just test the prefix of the class name?

Reply:
I think that should work now that I've drilled down to the `firstItem`. Let me update the test and just try using the `StyledBox__StyledBoxGap` portion.